src/index.html

```diff
@@ -1,14 +1,64 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap&subset=cyrillic" rel="stylesheet">
 </head>
+
 <body>
-  <h1>Moyo header</h1>
+  <header class="header">
+    <a href="#"><img src="/images/logo.png" alt="logo" class="img"></a>
+    <nav class="nav">
+      <ul class="nav-list">
+        <li class="nav-item active">
```

Comment:
```suggestion
        <li class="nav-item">
```
in don't see any `active` class in css